packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -191,10 +193,12 @@ export class RunningSummarizer implements IDisposable {
             : undefined;
 
     public handleSystemOp(op: ISequencedDocumentMessage) {
+        this.numSystemOps++;
         switch (op.type) {
             case MessageType.ClientLeave:
             case MessageType.ClientJoin:
             case MessageType.Propose: {
+                this.numNonSystemOps--; // Decrement to correct the counters
```

Comment:
Code before you changes looks wrong - I do not know why it looks only at some system ops.
Please use isSystemMessage() to figure out if message is system or not (it does not have ideal implementation as it uses opt-in vs. ideal out-out, but we will fix it separately)